app/src/main/res/values/strings.xml

```diff
@@ -714,6 +714,7 @@
     <string name="auto_device_theme_title">Automatic (device theme)</string>
     <string name="night_theme_summary">Select your favorite night theme — %s</string>
     <string name="select_night_theme_toast">You can select your favorite night theme below</string>
+    <string name="night_theme_available">This option is only available if Automatic (Device Theme) is selected for Theme</string>
```

Comment:
It might be good to insert the theme name dynamically by referencing `auto_device_theme_title`. This ensures that translators cannot mix things up by using different terms for the same setting. That's how we did it with `restricted_video`. However, we have not been consistent about this (e.g. see `show_play_with_kodi_title`).

Reply:
Thanks for the suggestion. I fixed it.